appengine/flexible/storage/main.py

```diff
@@ -57,12 +59,17 @@ def upload():
         content_type=uploaded_file.content_type
     )
 
+    # Make the blob public. This is not necessary if the
+    # entire bucket is public.
+    # See https://cloud.google.com/storage/docs/access-control/making-data-public.
+    blob.make_public()
+
     # The public URL can be used to directly access the uploaded file via HTTP.
     return blob.public_url
 
 
 @app.errorhandler(500)
-def server_error(e):
+def server_error(e: Union[flask.typing.GenericException, int]) -> str:
```

Comment:
This is throwing an Error, did you mean something else?

Reply:
Fixed. It looks like this is in the `main` branch for Flask but hasn't yet been released. :) https://github.com/pallets/flask/commit/6a4e7e948d7e5bd5b2b76c1b7f1a0392644bee1e